### PR TITLE
Migration to remove the legacy pulses

### LINKED ISSUE #32990: Pinning a pulse will hide it without any means of recovering it
### Describe the bug

1) Create a pulse in an old version (like 38.x), schedule it every hour
2) move to 46.x
3) pin it, so it dissapears

Now you'll keep getting the pulse, but you can't disable/delete it.

Workaround: go to the app db and set the collection_position of the pulse to "null"

how to identify the pulses? they don't have "dashboard_id" associated

### To Reproduce

Above

### Expected behavior

We should show dangling pulses so users can migrate them

### Logs

NA

### Information about your Metabase installation

```JSON
Does not matter, when we deprecated pulses in favor of subs this issue appeared
```

### Severity

P3 for all users not in cloud, for them it's a P2

### Additional context

scar tissue from pulse deprecation

### PR DESCRIPTION
**Pull request by qnkhuat**
PR to remove Pulse on FE: https://github.com/metabase/metabase/pull/35843
Epic https://github.com/metabase/metabase/issues/34147

Closes https://github.com/metabase/metabase/issues/32990 as pulse is no longer available

### CODE REVIEW

File: test/metabase/db/schema_migrations_test.clj [hunk 1]
```diff
@@ -732,8 +731,46 @@
...
+
+            dash-subscription-id (first (t2/insert-returning-pks! :pulse {:name            "A dashboard subscription"
+                                                                          :creator_id      user-id
+                                                                          :dashboard_id    dashboard-id
+                                                                          :collection_id   nil
+                                                                          :alert_condition "rows"
+                                                                          :parameters      "[]"
+                                                                          :created_at      :%now
+                                                                          :updated_at      :%now}))
+            legacy-pulse-id (first (t2/insert-returning-pks! :pulse {:name            "A leagcy pulse"
```
Comment (qnkhuat) [inline test/metabase/db/schema_migrations_test.clj:766]:
```suggestion
            legacy-pulse-id (first (t2/insert-returning-pks! :pulse {:name            "A legacy pulse"
```

File: test/metabase/db/schema_migrations_test.clj [hunk 2]
```diff
@@ -716,14 +715,14 @@
                                                                 :first_name "Metabase Internal User"
                                                                 :email "internal@metabase.com"
                                                                 :password (str (random-uuid))}))
-            original-db (t2/query {:datasource data-source} "SELECT * FROM metabase_database")
-            original-collections (t2/query {:datasource data-source}    "SELECT * FROM collection")
+            original-db-names (t2/select-fn-set :name :metabase_database)
```
Comment (calherries) [inline test/metabase/db/schema_migrations_test.clj:718]: Thank you for tidying this

File: test/metabase/db/schema_migrations_test.clj [hunk 3]
```diff
@@ -732,8 +731,46 @@
...
+                                                              :alert_condition "rows"
+                                                              :parameters      "[]"
+                                                              :created_at      :%now
+                                                              :updated_at      :%now}))
+            dashboard-id (first (t2/insert-returning-pks! :report_dashboard {:name       "A dashboard"
+                                                                             :creator_id user-id
+                                                                             :parameters "[]"
+                                                                             :created_at :%now
+                                                                             :updated_at :%now}))
+
```
Comment (calherries) [inline test/metabase/db/schema_migrations_test.clj:757]:
```suggestion
```

**Review by calherries — APPROVED** [review]
Very solid stuff

**Review by calherries — APPROVED** [review]
Still looking good

**Review by luizarakaki — APPROVED** [review]

**Pull request merged**